Packs/CommonScripts/ReleaseNotes/1_11_63.md

```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### StixCreator
+
+##### StixCreator
+
+- Added an option to enter a flag for creating SCO indicators.
+- Updated the process of generating IDs for SDO indicators.
```

Comment:
The update part is the use of create_sdo_stix_uuid function to create the stix_id of the SDO indicators or SCO indicators when the flag is false. I did this update because I discovered that in each click on the button "export (stix)" generates a new indicator ID. After I talked to @Ni-Knight we decided to do it this. It is also written as a comment in the issue.